gallery/src/components/NxTabs/NxTabsModalExample.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+const NxTabsModalExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState<number>(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <>
+      <button onClick={() => setShowModal(true)} className="nx-btn">Open Modal</button>
```

Comment:
I'd think, when in react component examples, we should use `NxButton` instead of `<button>`.  The other modal examples have already been updated in that way.

Reply:
> I'd think, when in react component examples, we should use `NxButton` instead of `<button>`. The other modal examples have already been updated in that way.

I agree, bad copy/pasta on my part: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/cda6e6e96355d6fbfe70cdbc3722c0e6835f934a

Reply:
> I agree, bad copy/pasta on my part: cda6e6e

You missed some, including the one this comment is on